step 2

alkor63 opened the pull request.

alkor63 committed:
step 2

alkor63 committed:
step 2

alkor63 committed:
step 2

alkor63 committed:
step 2

alkor63 committed:
step 3

alkor63 committed:
step 3

alkor63 committed:
step 4
Готово !

DmitryN10 commented on src/AutoRacing/DriverB.java:
```diff
@@ -0,0 +1,46 @@
+import java.util.Objects;
+@Setter
+@Getter
+public class DriverB {
```
public class DriverB extends Driver<Car>

DmitryN10 replied:
Сам наверное замечаешь, что у тебя все 3 класса одинаковые. Как раз для этого нужен полиморфизм, чтобы создать шаблонный класс и переиспользовать.
Ниже прикрепил хороший пример как должно быть
https://github.com/lisichkaNataly/OOP_4.Polymorphism/pull/1/files#diff-9e26822aade29fb4192cc90d6b24ba8b4cbad852bf934ef12d28dacf5c919feb

alkor63 committed:
step 4
Готово !

alkor63 committed:
step 4
Получается, либо полиморфизм с категориями драйверов либо дженерики и класс Driver

alkor63 committed:
step 4
Готово !
